cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -69,40 +69,35 @@
 
     <div class="content-container">
         <div class="bg-gray-100 border-b flex flex-wrap text-center px-5">
-            <div class="flex items-center ml-3 py-4">
+            <div class="flex flex-col items-start ml-3 py-4 flex-grow">
                 <div class="text-base font-bold text-gray-900">DNA Feature Aliases:</div>
-                {% comment %}
-                We want to line break wherever we can for narrow screens if the feature might have
-                a weird, long "name" like gRNAs and DHSs do -- basically anything that is a '"source type"
-                We can identify any of these "source types" because they (and only they) don't have ensembl ids
-                {% endcomment %}
```

Comment:
The "break-all" class is still there, so I think the comment is still warranted.